src/FunctionApps/python/IntakePipeline/__init__.py

```diff
@@ -35,7 +35,7 @@ def run_pipeline(
     message: str,
     message_mappings: Dict[str, str],
     fhir_url: str,
-    access_token: str,
+    cred_manager: AzureFhirServerCredentialManager,
```

Comment:
From @bamader: As above, import will now be from azure.py

Response by @spencer-skylight: This has been addressed.